<?xml version="1.0"?>
<launch>
    <!-- 启动底盘驱动 -->
    <include file="$(find autolabor_pro1_driver)/launch/driver.launch" />
    
    <!-- 启动USB摄像头 -->
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>
    </node>
    
    <!-- 启动跟踪节点 -->
    <node name="tracker_node" pkg="track_pkg" type="runtracker" output="screen">
        <param name="camera_topic" value="/usb_cam/image_raw" />
        <param name="linear_speed" value="0.2" />
        <param name="angular_speed" value="0.2" />
    </node>
</launch> 